src/python/pants/scm/git.py

```diff
@@ -73,7 +73,7 @@ def clone(cls, repo_url, dest, binary='git'):
     return cls(binary=binary, worktree=dest)
 
   @classmethod
-  def _invoke(cls, cmd):
+  def _invoke(cls, cmd, stderr=None):
```

Comment:
I think this should be `subprocess.DEVNULL`. Is there a particular reason you want wirh `None`?

Reply:
I don't think we always want to suppress stderr when calling `_invoke`, just in the specific case where we call it in `detect_worktree` to run `git rev-parse`. So the default parameter in `_invoke` should do the same thing as calling `Popen` without `stderr` at all, unless we tell it otherwise.

Reply:
I see, that makes sense.

Could you please add a docstring entry explaining that if left as `None`, `subprocess` will keep its default of printing to stderr? That was my main confusion, being unsure what `None` does in this context.

Reply:
Updated the docstring to clarify this